src/main/java/seedu/address/model/date/Date.java

```diff
@@ -80,13 +79,22 @@ public static boolean isValidDateValue(String test) {
     }
 
     /**
-     * Returns true if the {@code LocalDate} specified is after the current date, false otherwise.
+     * Returns true if a given string is after the current date.
+     * @param test string input, the format check for test should be done before calling this function.
      */
-    public static boolean isAfterCurrentDate(String date) {
-        LocalDate dateToCheck = LocalDate.parse(date, logFormatter);
+    public static boolean isAfterCurrentDate(String test) {
+        LocalDate dateToCheck = LocalDate.parse(test, logFormatter);
         return dateToCheck.isAfter(LocalDate.now());
     }
 
+    /**
+     * Returns the integer age of a person.
```

Comment:
Not sure if we should include `person` in the header comment since this class is not exclusively used for persons? It also used in events.

Do you think this works better?

```suggestion
     * Returns the integer age of this Date object.
```

Reply:
Hmm I think it's ok to include `person` in the header comment since this method is specific to `Person` object, but it's a bit unclear. How about the change below?
```suggestion
     * Returns the integer age of this Date object when the Date Object is used as date of birth of a person.
```